src/components/button/_button-base.scss

```diff
@@ -107,7 +107,7 @@ $md-mini-fab-padding: 8px !default;
   height: $size;
   padding: 0;
 
-  i {
+  md-icon {
```

Comment:
This should actually be `i`, not `md-icon`.  `md-icon` is already aligned correctly; it's just regular `i` that need the padding.